### ISSUE TITLE
TFS error TF400019 error prompt points to this page, but this page has no help whatever to fix the error

### ISSUE THREAD

**Issue by markyoung2u** [MicrosoftDocs/azure-devops-docs#11916]
[Enter feedback here]
tf vc stat
TF400019: The pending changes table for the local workspace W103KLCN53;Young, Mark A could not be opened. Exception of type 'Microsoft.TeamFoundation.VersionControl.Client.InvalidPendingChangeTableException' was thrown.

I locked up my notebook and had to hold power button down to force power off to recover. Now I cannot access any workspace, checkin, etc. Google fixes are basically backup your files, delete and recreate your workspace. I cannot delete this workspace from inside VS - it gets the same error.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: af8d20f6-f3e6-800d-72bd-2f006a4f6bc9
* Version Independent ID: 84e393f0-a3b5-89cd-faee-7d25594a3d4d
* Content: [Create and work with workspaces - Azure Repos](https://docs.microsoft.com/en-us/azure/devops/repos/tfvc/create-work-workspaces?view=azure-devops&redirectedfrom=MSDN&f1url=%3FappId%3DDev15IDEF1%26l%3DEN-US%26k%3Dk(vs.tfc.sourcecontrol.DialogManageWorkspaces)%26rd%3Dtrue&viewFallbackFrom=vsts)
* Content Source: [docs/repos/tfvc/create-work-workspaces.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/repos/tfvc/create-work-workspaces.md)
* Product: **devops**
* Technology: **devops-code-tfvc**
* GitHub Login: @MaximMalygin
* Microsoft Alias: **v-maksim**

**Comment by TimShererWithAquent** [comment]
@markyoung2u - Thanks for your question. It looks like you're working through an issue with your scenario or implementation, rather than an issue with the documentation. Please consider these resources:

- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

**Comment by markyoung2u** [comment]
True, this is an issue but the doc is lacking and I spent all weekend trying to resolve. Here is what fixed the issue, not even 1/10th of all the things I tried. It is important this page provide something useful although I know MS rarely details any kind of problem resolution. At stake is losing hours or days of coding you have not checked in. Even backing up this many files can be problematic.
Close VS and any app accessing any local workspace file, or else next step will get "access denied" - can't stress this enough.
Rename the local workspace dir: e.g. my local workspace is c:\tc2\s so:
  cd \tc2
  move s sSave
Now try to access your workspace in VS and it should work. Since your files are no longer where TFS expects them to be, get the latest files (to update c:\tc2\s tree). Use Beyond Compare or other diff utility to compare the s and sSave trees to recreate any changes you were working on before the error.
